content/community/events.md

```diff
@@ -14,8 +14,8 @@ g3Upcoming:
     Description:  "The Gen3 Community Forum consists of those developing and operating Gen3 data commons and data meshes, those considering developing Gen3 data commons and meshes, and other stakeholders involved with the Gen3 community. The four aims of the Forum are: to share knowledge about Gen3, its architecture, and the Gen3 roadmaps and priorities; to strengthen the connection between the Gen3 core team and those developing, operating and using Gen3 platforms; to design a set of ongoing community engagement activities; and to discuss and agree on key shared development priorities between the Gen3 core team and the Gen3 community. The virtual community forum is co-hosted by the University of Chicago and the Australian BioCommons."
     Image: /figs/gen3_new_logo.png
     Link: https://www.eventbrite.com/e/gen3-community-forum-2022-tickets-384218496867
-    LocalLink:  /community/events/gen3forum_202210
-    LocalLink2: /community/events/gen3forum_2022_agenda
+    LocalLink2:  /community/events/gen3forum_202210
+    LocalLink: /community/events/Gen3workshop
```

Comment:
according to the Ci logs, this was deployed to the alpha website. But i can't find the page, these links don't work:
- http://alpha.gen3.org/community/events/Gen3workshop
- http://alpha.gen3.org/community/events/Gen3Workshop

Reply:
Found it, but it doesn't look good: http://alpha.gen3.org/community/events/gen3workshop/index.html
@smvgarcia